src/wizard/wizardStep/wizardStep.html

```diff
@@ -9,30 +9,30 @@
 
 	<div class="wizard-btn-container modal-footer"
          data-ng-hide="getCurrentStep() === 1 && stepCount === 1 && !cancelButton && !confirmButton">
-		<button class="btn btn-default"
+		<button class="btn btn-default" type="button"
+                data-ng-bind="wizardCancelButtonText"
 				data-ng-click="onCancel()"
                 data-ng-show="cancelButton">
-            <span data-ng-bind="wizardCancelButtonText"></span>
 		</button>
 
-		<button class="btn"
+		<button class="btn" type="button"
```

Comment:
`btn-default` maybe?

Reply:
@antleblanc Nope, not in this case (exception)